runtime/start.py

```diff
@@ -30,6 +30,9 @@
     "domain": domain,
     "auth_audience": "",
     "auth_self_signed_jwk": "",
+    "CONDA_DEFAULT_ENV": "plots-faas",
```

Comment:
why do we need this stuff?

Reply:
so that we don't have to do `os.system('opt/mamba/envs/plots-faas/bin pip install ...')` and just `os.system('pip install ...)`